src/module_1/module_1_meteo_api.py

```diff
@@ -3,34 +3,45 @@
 import logging
 import time
 import sys
-import json
-import numpy as np
+import pandas as pd
 import matplotlib.pyplot as plt
-import pickle 
+import urllib3
+# import json
+# import numpy as np
+
+
+# Disable the InsecureRequestWarning due to using verify=False from corporate laptop
+urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
+
 
 """Define global variables"""
+
 API_URL = "https://climate-api.open-meteo.com/v1/climate?"
 
 COOL_OFF_TIME = 5
 MAX_RETRY_ATTEMPTS = 3
 
 COORDINATES = {
-"Madrid": {"latitude": 40.416775, "longitude": -3.703790},
```

Comment:
Use a Python linter in VS code which should handle this automatically

Reply:
Noted down (run `make lint` from cmd for theses cases)